wordpress-mobile/WordPress-Android
# Blaze: Rename feature entry point tracking events

Fixes #18034 

This PR renames the following tracking events from > to
- `blaze_feature_displayed` to `blaze_entry_point_displayed`
- `blaze_feature_tapped` to `blaze_entry_point_tapped`
- `blaze_feature_menu_accessed` to `blaze_entry_point_menu_accessed`
- `blaze_feature_hide_tapped` to `blaze_entry_point_hide_tapped`


To test:
- Install and launch the app
- Login with a wp.com 
- Select a site that is eligible for blaze
- Navigate to Me > App Settings > Privacy
- Enable collection information
- Navigate to Me > App Settings > Debug Settings
- Enable blaze and restart
- Navigate to My Site tab
- ✅ Verify the log contains 🔵 Tracked: blaze_entry_point_displayed, Properties: {"source":"dashboard_card"}
- Tap on the Blaze dashboard card
- ✅ Verify the log contains 🔵 Tracked: blaze_entry_point_tapped, Properties: {"source":"dashboard_card"}
- Navigate back to the My Site tab
- Tap on the More menu within the Blaze Dashboard card
- ✅ Verify the log contains🔵 Tracked: blaze_entry_point_menu_accessed, Properties: {"source":"dashboard_card"}
- Tap on the "hide this" menu option 
- ✅ Verify the log contains🔵 Tracked: blaze_entry_point_hide_tapped, Properties: {"source":"dashboard_card"}
**Note**: Once you hide the blaze dashboard card, it will not reshow until you logout and log in again


## Regression Notes
1. Potential unintended areas of impact
Blaze event collection

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
